api/v01/entities/vrp_input.rb

```diff
@@ -465,11 +465,11 @@ module VrpVehicles
     optional(:cost_fixed, type: Float, desc: 'Cost applied if the vehicle is used')
     optional(:cost_distance_multiplier, type: Float, desc: 'Cost applied to the distance performed')
 
-    optional(:cost_time_multiplier, type: Float, desc: 'Cost applied to the total amount of time of travel (Jsprit) or to the total time of route (ORtools). Not taken into account within periodic heuristic.')
+    optional(:cost_time_multiplier, type: Float, desc: 'Cost applied to the total time of route (ORtools). Not taken into account within periodic heuristic.')
     optional(:cost_value_multiplier, type: Float, desc: 'Multiplier applied to the value matrix and additional activity value. Not taken into account within periodic heuristic.')
     optional(:cost_waiting_time_multiplier, type: Float, desc: 'Cost applied to the waiting time in the route. Not taken into account within periodic heuristic.')
     optional(:cost_late_multiplier, type: Float, desc: 'Cost applied if a point is delivered late (ORtools only). Not taken into account within periodic heuristic.')
-    optional(:cost_setup_time_multiplier, type: Float, desc: 'Cost applied on the setup duration (Jsprit only). Not taken into account within periodic heuristic.')
+    optional(:cost_setup_time_multiplier, type: Float, desc: 'DEPRECATED : Cost applied on the setup duration. Not taken into account within periodic heuristic.')
```

Comment:
For these kind of (previously Jsprit only parameters), we can even change the description to ` 'DEPRECATED : Jspirit solver and related parameters are not supported anymore.` otherwise the description clutters the documentation for no gain. What do you think @braktar ?